### PR TITLE
Adding a results-embed.html that includes the classification.js inline

### PR DESCRIPTION
**Pull request by edcdavid**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -136,7 +136,7 @@ get-db:
 	docker pull ${OCT_IMAGE}
 	docker run -v ${REPO_DIR}/offline-db:/tmp/dump:Z --user $(shell id -u):$(shell id -g) --env OCT_DUMP_ONLY=true ${OCT_IMAGE}
 delete-db:
-	rm -rf ${REPO_DIR}/offline-db
+	rm -rf ${REPO_DIR}/offline-dbed -i -e 's@  <script src="./classification.js"></script>@@g' script/results1.html
```
Comment (sebrandon1) [inline Makefile:139]: Is this supposed to be changed?

Comment (edcdavid) [reply Makefile]: good catch! fixed

**Review by sebrandon1 — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -151,5 +151,8 @@ build-image-tnf:
 		-t ${REGISTRY}/${TNF_IMAGE_NAME}:${TNF_VERSION} \
 		-f Dockerfile .
 
+# Generates the classification.js file and creates anew result-embed.html
```
Comment (rdavid) [inline Makefile:154]: Space is needed: a new

File: script/results-embed.html [hunk 3]
```diff
@@ -0,0 +1,1272 @@
...
+        }
+      }
+      const clonedTable = table.cloneNode(true);
+      body.appendChild(clonedTable);
+    }
+
+  </script>
+</body>
+
+</html>
```
Comment (rdavid) [inline script/results-embed.html:1272]: Add a new line to the end of a file.

File: Makefile [hunk 4]
```diff
@@ -151,5 +151,8 @@ build-image-tnf:
...
 classification-js:
 	./tnf generate catalog javascript > script/classification.js
+	sed '/<script src=".\/classification.js"><\/script>/e echo "<script>";cat  script/classification.js; echo "<\/script>"' script/results.html > script/results-embed.html
```
Comment (rdavid) [inline Makefile:157]: Space is not needed after redirect `>/dev/null`. I think `shfmt` complains.

Comment (rdavid) [inline Makefile:157]: Put every command on a new line, `;` is not needed.

File: Makefile [hunk 5]
```diff
@@ -151,5 +151,8 @@ build-image-tnf:
 		-t ${REGISTRY}/${TNF_IMAGE_NAME}:${TNF_VERSION} \
 		-f Dockerfile .
 
+# Generates the classification.js file and creates anew result-embed.html
 classification-js:
 	./tnf generate catalog javascript > script/classification.js
```
Comment (rdavid) [inline Makefile:156]: Space is not needed after redirect >.

**Review by rdavid — APPROVED** [review]

File: Makefile (hunk 4 above)
Comment (edcdavid) [reply Makefile:157]: the ";" is used in this case because the 3 commands are executed by sed

Comment (rdavid) [reply Makefile]: You're right. For better formatting, add space after `;`, and remove space after `cat`.

**Review by rdavid — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (edcdavid) [reply Makefile]: fixed

File: script/results-embed.html (hunk 3 above)
Comment (edcdavid) [reply script/results-embed.html]: fixed

File: Makefile (hunk 4 above)
Comment (edcdavid) [reply Makefile]: fixed

File: Makefile (hunk 5 above)
Comment (edcdavid) [reply Makefile]: fixed

File: Makefile (hunk 4 above)
Comment (edcdavid) [reply Makefile]: fixed

**Comment by rdavid** [comment]
Good to go! Thanks!

**Pull request merged**